packages/docs/components/link.md

```diff
@@ -71,32 +66,16 @@ Odyssey has removed unique styling for `:visited` links. This is an intentional
   <li>Try to limit a link to at most 3 words</li>
   <li>Choose link text that describes the destination (e.g. "Settings"), rather than generic text (e.g. "Click here" or a URL)</li>
   <li>Avoid using a link <code>&lt;a&gt;</code> for actions; use a button <code>&lt;button&gt;</code> instead</li>
-  <li>Avoid underlining text for purely decorative purposes, as it will be mistaken for linked text</li>
 </ul>
 
-### Styling
-
-By default, links embedded inside blocks of prose text are underlined, while links used inside navigational menus and nav bars do not require an underline and may be custom-styled.
-
-Link styling can also be adjusted for branding, theme, or contextual purposes. Examples include color, font size, or font weight. Be mindful of consistency with the system you are designing for.
-
 ### Accessibility
 
-Links should display a visible <code>:focus</code> state when users interact via keyboard.
+Links in Odyssey are not underlined, but do maintain a minimum 3:1 contrast ratio with our body text color and a 4.5:1 contrast ratio with our available background colors. If you deviate from these standards via themeing, please ensure that your links have a non-color indicator, e.g. an underline.
```

Comment:
Odyssey doesnt provide theming, I would use different language here to avoid confusion.

Reply:
Updated to "overrides"!